src/Kernel/Magic/Simulate.cs

```diff
@@ -101,9 +101,34 @@ public async Task<ExecutionResult> RunAsync(string input, IChannel channel)
             var maxNQubits = 0L;
 
             using var qsim = new QuantumSimulator()
-                .WithJupyterDisplay(channel, ConfigurationSource)
+                //.WithJupyterDisplay(channel, ConfigurationSource)
                 .WithStackTraceDisplay(channel);
-            qsim.OnDisplayableDiagnostic += channel.Display;
+
+            ////
+            qsim.DisableLogToConsole();
+            qsim.OnLog += channel.Stdout;
+            ////
+
+            //qsim.OnDisplayableDiagnostic += channel.Display;
+
+            ////
+            qsim.OnDisplayableDiagnostic += (displayable) =>
```

Comment:
Goodness, this is a lot nicer than having the whole 300 line file to work around the different ways of reporting dumps, thanks for doing this!